test/sandbox/contract/printEncodedFunctions.ts

```diff
@@ -0,0 +1,23 @@
+import { web3 } from '../../../src/helpers/web3'
+import Logger from '../../../src/helpers/Logger'
+import { AbiItem } from 'web3-utils'
+
+require('dotenv').config()
+
+/**
+ *  SANDBOX: Print the encoded version for the functions
+ *  RUN:     yarn sandbox test/sandbox/printEncodedFunctions.ts
+ */
+const log = new Logger('sandbox:printEncodedFunctions')
+
+async function exec (): Promise<void> {
```

Comment:
Why is this asynchronous?

Reply:
same as other comment. I'll resolve this one so we discuss there